permits/views.py

```diff
@@ -496,24 +498,34 @@ def permit_request_actors(request, permit_request_id):
 
 @login_required
 def permit_request_geo_time(request, permit_request_id):
-
     permit_request = services.get_permit_request_for_user_or_404(request.user, permit_request_id)
+    PermitRequestGeoTimeFormSet = modelformset_factory(
+        models.PermitRequestGeoTime,
+        form=forms.PermitRequestGeoTimeForm,
+        extra=0,
+        min_num=1,
+        can_delete=True
```

Comment:
Appears as a checkbox in form. Any simple mean to make it look like something more obvious for the user ?

Reply:
No simple way I’m afraid, that’s the way Django does it by default with fieldsets. We could replace this by something else, but it’s not that simple.

Reply:
Ok, no worries